runtime/development/Cargo.toml

```diff
@@ -124,6 +124,7 @@ pallet-connectors-gateway = { path = "../../pallets/connectors-gateway", default
 pallet-crowdloan-claim = { path = "../../pallets/crowdloan-claim", default-features = false }
 pallet-crowdloan-reward = { path = "../../pallets/crowdloan-reward", default-features = false }
 pallet-data-collector = { path = "../../pallets/data-collector", default-features = false }
+pallet-ethereum-allowlist = { path = "../../pallets/ethereum-allowlist", default-features = false }
```

Comment:
Here is the issue I think, you need to add this pallet to std/runtime-benchmarks/try-runtime

Reply:
Yup, I already have those changes locally and it's fine, I'm tackling the clippy issue now.